gradle.properties

```diff
@@ -1,5 +1,5 @@
 group=org.partiql
-version=0.6.1-SNAPSHOT
+version=0.6.1
```

Comment:
nit: could also update the `README` example version to 0.6.1. See https://github.com/partiql/partiql-ir-generator/blob/d39fb6a9f841454db3c9594b99b892ec2ad6ffa3/README.md?plain=1#L82